core/src/test/resources/test_tumor_type_summaries.txt

```diff
@@ -65,32 +65,32 @@ BRAF	MKRN1-BRAF Fusion	Breast Cancer	BRAF, an intracellular kinase, is frequentl
 BRAF	V600K	Non-Small Cell Lung Cancer	BRAF, an intracellular kinase, is frequently mutated in melanoma, thyroid and lung cancers among others.	The BRAF V600K mutation is known to be oncogenic.	While the RAF-targeted inhibitor dabrafenib in combination with the MEK1/2-targeted inhibitor trametinib is FDA-approved for the treatment of patients with BRAF V600E mutant non-small cell lung cancer (NSCLC), this combination is not specifically approved for the treatment of patients with BRAF V600K mutant NSCLC. However, laboratory data demonstrate that cells expressing BRAF V600K respond similarly to dabrafenib + trametinib as cells with BRAF V600E, and patients with BRAF V600K mutant melanoma derived similar clinical benefit to dabrafenib + trametinib as those with BRAF V600E.
-BRAF	V600M	Non-Small Cell Lung Cancer	BRAF, an intracellular kinase, is frequently mutated in melanoma, thyroid and lung cancers among others.	The BRAF V600M mutation is likely oncogenic.	The RAF-targeted inhibitor dabrafenib in combination with the MEK1/2-targeted inhibitor trametinib is FDA-approved for the treatment of patients with BRAF V600E mutant non-small cell lung cancer.
-BRAF	V600R	Non-Small Cell Lung Cancer	BRAF, an intracellular kinase, is frequently mutated in melanoma, thyroid and lung cancers among others.	The BRAF V600R mutation is known to be oncogenic.	The RAF-targeted inhibitor dabrafenib in combination with the MEK1/2-targeted inhibitor trametinib is FDA-approved for the treatment of patients with BRAF V600E mutant non-small cell lung cancer.
-BRAF	V600D	Non-Small Cell Lung Cancer	BRAF, an intracellular kinase, is frequently mutated in melanoma, thyroid and lung cancers among others.	The BRAF V600D mutation is known to be oncogenic.	The RAF-targeted inhibitor dabrafenib in combination with the MEK1/2-targeted inhibitor trametinib is FDA-approved for the treatment of patients with BRAF V600E mutant non-small cell lung cancer.
-BRAF	V600G	Non-Small Cell Lung Cancer	BRAF, an intracellular kinase, is frequently mutated in melanoma, thyroid and lung cancers among others.	The BRAF V600G mutation is likely oncogenic.	The RAF-targeted inhibitor dabrafenib in combination with the MEK1/2-targeted inhibitor trametinib is FDA-approved for the treatment of patients with BRAF V600E mutant non-small cell lung cancer.
+BRAF	V600M	Non-Small Cell Lung Cancer	BRAF, an intracellular kinase, is frequently mutated in melanoma, thyroid and lung cancers among others.	The BRAF V600M mutation is likely oncogenic.	While the RAF-targeted inhibitor dabrafenib in combination with the MEK1/2-targeted inhibitor trametinib is FDA-approved for the treatment of patients with BRAF V600E mutant melanoma, non-small cell lung cancer and anaplastic thyroid cancer, the clinical utility of dabrafenib in combination with trametinib in patients with BRAF V600M mutant non-small cell lung cancer has yet to be defined.
```

Comment:
this is getting the summary under V600, Other tumor types instead of V600 NSCLC....I would think it should get the summary under NSCLC

Reply:
same for V600R, D, G below